src/core/components/rerender/Rerender.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Rerender } from './Rerender';
+
+const meta: Meta<typeof Rerender> = {
+    title: 'Core/Components/Rerender',
+    component: Rerender,
+};
+
+type Story = StoryObj<typeof Rerender>;
+
+/**
+ * Default usage example of the Rerender component.
+ */
+export const Default: Story = {
+    render: (args) => (
+        <Rerender intervalDuration={args.intervalDuration}>{(time) => <div>Current Time: {time}</div>}</Rerender>
```

Comment:
It would be nice to format the date through the formatter and display a formatted date with seconds

Reply:
Yes, but I don't think we have a format that would display seconds, the `YEAR_MONTH_DAY_TIME` has a precision to a minute.

Reply:
Mm.. we could use the relative format and display the seconds as: 1 second ago, 2 seconds ago, ...

Reply:
<img width="1024" alt="image" src="https://github.com/aragon/ods/assets/172602066/29024aec-087f-4a57-85f9-2ee576c87c70">
It would then stuck to being _0 seconds ago_ since it's the current time relative to current time. Used `new Date(time).toISOString()`, I think that should be enough 😅

Reply:
Yes works fine thanks! 😄